DEVELOPMENT.adoc

```diff
@@ -0,0 +1,180 @@
+* link:https://www.python.org/downloads/[Python 3.7.8]
+* Xcode 12.5
+
+== Installation
+
+=== Install Java & Kotlin
+
+You could use link:https://brew.sh/[Homebrew] to install on mac
+
+[source,bash]
+----
+brew tap adoptopenjdk/openjdk
+brew install adoptopenjdk8
+brew install kotlin
+----
+
+OR The easiest way to install Java and Kotlin is to use link:https://sdkman.io/[SDK MAN]
+
+Just open your terminal and enter:
+
+[source,bash]
+----
+curl -s "https://get.sdkman.io" | bash
+----
+
+Follow the setup instructions and verify that your installation works:
+
+[source,bash]
+----
+sdk version
+----
+
+If it shows you something like `5.11.5+713` then your good to continue.
+
+Just install the mentioned dependencies:
+
+[source,bash]
+----
+sdk install java 8.0.292.hs-adpt
+sdk install kotlin 1.4.32
+----
+
+Check your installation by:
+
+[source,bash]
+----
+java -version
+kotlin -version
+----
+
+=== Install Python
+
+It is recommended to use link:https://github.com/pyenv/pyenv[PyEnv] for managing dedicated python versions.
+
+Use link:https://brew.sh/[Homebrew] to install it on mac:
+
+[source,bash]
+----
+brew install pyenv
+----
+
+Make sure you are in the project root and install the correct Python version which is derived from `.python-version` file:
+
+[source,bash]
+----
+pyenv install
+----
+
+=== Android Studio
+
+For better interop with iOS you may install the link:https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile[KMM plugin for Android Studio]]. It let's you run, test and debug shared code on Android and iOS without switching the IDEs.
+
+=== First time
+
+As the project includes a Git submodule, this must be initialised on first use
+
+[source,bash]
+----
+git submodule update --init --recursive
+----
+
+== Development Process
+
+=== Features
+
+Every change has to branch of from `main` and use this branch naming convention:
+
+* `feature/{type_of_change}-{short_description}` or with ticket id `feature/{ticket_id}/{type_of_change}-{short_description}`
+
+`main` must be always in releasable state.
+
+==== Type Of Change
+
+- *add* for new features or functionality
+- *change* for changes in existing features or functionality
+- *remove* | for removed features or functionality
+- *fix* for any bug fixes
+- *bump* for dependency updates
+- *security* in case of vulnerabilities
+
+Examples:
+
+- `feature/NAT-456/add-awesome-hashing-algorithm`
+- `feature/add-awesome-hashing-algorithm`
+- `feature/remove-not-so-awesome-algorithm`
+- `feature/fix-algorithm-corner-case`
+- `feature/bump-lib-to-1.3.0`
+
+== Release
+
+A release branches of from `main` branch with following pattern:
+
+* `release/{major}.{minor}(.{patch})/prepare-{major}.{minor}.{patch}` while patch is optional
+
+Add following changes:
+* [ ] Update CHANGELOG.adoc by creating a new Unreleased section and change current unreleased to release version
+* [ ] Update README.adoc `fhir-sdk-version` to release version
```

Comment:
Should that be a checkbox list or is this correct displayed?

Reply:
Should have been a checkbox, but will remove it